modules/apps/account/account-admin-web/src/main/resources/META-INF/resources/account_entries_admin/js/AccountEntriesAdminPortlet.es.js

```diff
@@ -52,10 +54,10 @@ class AccountEntriesAdminPortlet extends PortletBase {
 	 */
 	updateVisibility_(typeSelect) {
 		if (typeSelect.value === 'business') {
-			dom.removeClasses(this.businessAccountOnlySection, 'hide');
+			this.businessAccountOnlySection?.classList.remove(HIDE_CSS_CLASS);
 		}
 		else {
-			dom.addClasses(this.businessAccountOnlySection, 'hide');
+			this.businessAccountOnlySection?.classList.add(HIDE_CSS_CLASS);
```

Comment:
Maybe this would all be 

```javascript
this.businessAccountOnlySection?.classList.remove('hide', typeSelect.value === 'business');
```

Reply:
It worked well when replacing `remove` by `toggle` function :p

Thanks! It will reduce dozens of LOC :)